RecoParticleFlow/PFProducer/python/particleFlow_cfi.py

```diff
@@ -70,12 +70,22 @@
     maxEeleOverPout = cms.double(0.2),
     maxDPhiIN = cms.double(0.1)    
     ),
+    electron_protectionsForBadHcal = cms.PSet(
+    full5x5_sigmaIetaIeta = cms.vdouble(0.0126, 0.0457), # EB, EE
+    eInvPInv = cms.vdouble(0.209, 0.132),
+    dEta = cms.vdouble(0.01, 0.02),
+    dPhi = cms.vdouble(0.148, 0.19),
```

Comment:
one more comment on these, I think they are on the loose side. For e/gamma, we dont really care. JetMET might. 

However if the intended strategy is to do a first pass with a loose selection and see how much it really causes problems for jetMET, that is fine for me to try this.

You see here that for the proposed selection, it kills the low pt fake electrons but the problem is the high pt chaps, those will start messing around with the jets.  This is the JetHT relval sample

![image](https://user-images.githubusercontent.com/5021403/43950646-2a4d38e2-9c91-11e8-8d08-9e9b10f35cb4.png)

Now if you tightened a bit more : sigmaIEtaiEta<0.031, dEtaIn<0.01, dPhiIn<0.1, 1/E-1/p < 0.132, you get
![image](https://user-images.githubusercontent.com/5021403/43950808-aeffbe0c-9c91-11e8-8bc3-18be08c4cb0b.png)

and you still have  a high pt problem. To really kill it you can put a track isolation < 5 GeV cut on it. 
![image](https://user-images.githubusercontent.com/5021403/43950871-e05bd756-9c91-11e8-998b-74b54cbd7744.png)

All this really depends on what jetMET need here, its them not e/gamma that will have the problem here